test-platform-api/prisma/schema.prisma

```diff
@@ -3,16 +3,18 @@ generator client {
 }
 
 datasource db {
-  provider  = "postgresql"
-  url       = env("DATABASE_URL")
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
 }
 
 model Question {
   id        Int          @id @default(autoincrement())
   createdAt DateTime     @default(now())
-  updatedAt DateTime     @updatedAt
+  updatedAt DateTime?    @updatedAt
   sentence  String
-  answer    String?
+  category  String
+  answer    String[]
+  options   String[]
```

Comment:
1. What is the distinction between the 'answer' and 'options' properties? 
2. We appear to be missing the correct answer(s) attribute ?
3. I suppose category is `categories`, because one question will belong to several categories. Could you consider which I suppose ?
Thanks @khanh-nguyen-saigontechnology-com.

Reply:
@datngo21 
1. `options` is array list of selection options
2. correct answer(s) is `answers`
3. for now question only belong to 1 category. It's based on example json file. We will enhance it later.